src/Core/src/Platform/iOS/MauiTextView.cs

```diff
@@ -112,6 +112,17 @@ public override NSAttributedString AttributedText
 				}
 			}
 		}
+		
+		public override NSAttributedString? AccessibilityAttributedValue
```

Comment:
Would require changes also in the Catalyst public API txt
```
D:\a\_work\1\s\src\Core\src\Platform\iOS\MauiTextView.cs(118,4): error RS0016: Symbol 'AccessibilityAttributedValue.get' is not part of the declared public API (https://github.com/dotnet/roslyn-analyzers/blob/main/src/PublicApiAnalyzers/PublicApiAnalyzers.Help.md) [D:\a\_work\1\s\src\Core\src\Core.csproj::TargetFramework=net9.0-maccatalyst18.0]
D:\a\_work\1\s\src\Core\src\Platform\iOS\MauiTextView.cs(124,4): error RS0016: Symbol 'AccessibilityAttributedValue.set' is not part of the declared public API (https://github.com/dotnet/roslyn-analyzers/blob/main/src/PublicApiAnalyzers/PublicApiAnalyzers.Help.md) [D:\a\_work\1\s\src\Core\src\Core.csproj::TargetFramework=net9.0-maccatalyst18.0]
    22 Warning(s)
    2 Error(s)

```

Reply:
RIght! Thanks!